packages/rspack-test-tools/tests/configCases/library/modern-module-named-import-externals/rspack.config.js

```diff
@@ -0,0 +1,85 @@
+module.exports = {
+	mode: "none",
+	entry: { main: "./index.js", test: "./test.js" },
+	output: {
+		module: true,
+		library: {
+			type: "modern-module"
+		},
+		filename: "[name].js",
+		chunkFormat: "module"
+	},
+	experiments: {
+		outputModule: true
+	},
+	resolve: {
+		extensions: [".js"]
+	},
+	externalsType: "module",
+	externals: [
+		"externals0",
+		"externals1",
+		"externals2",
+		"externals3",
+		"externals4"
+	],
+	optimization: {
+		concatenateModules: true,
+		usedExports: true
+	},
+	plugins: [
+		function () {
+			const handler = compilation => {
+				compilation.hooks.processAssets.tap("testcase", assets => {
+					const source = assets["test.js"].source();
+					expect(source).toMatchInlineSnapshot(`
+				import { a, b } from "externals0";
+				import { a as a_0 } from "externals1";
+				import default_0 from "externals2";
```

Comment:
The source code here before transform is

```js
import defaultValue from 'externals2'
```

Could we reserve the `defaultValue` and append index suffix to de-conflict, it will be much better for readability when default import is heavily used (just like how esbuild behaves now [esbuild - Try in the browser](https://esbuild.github.io/try/#YgAwLjI1LjIALS1idW5kbGUgLS1leHRlcm5hbDpsb2Rhc2ggLS1mb3JtYXQ9ZXNtAGUAZW50cnkuanMAaW1wb3J0IGxvZCBmcm9tICdsb2Rhc2gnCmltcG9ydCB7IHN1YiB9IGZyb20gJy4vZmlsZS5qcycKCmxvZC5hZGQoKQpzdWIoKQAAZmlsZS5qcwBpbXBvcnQgbG9kIGZyb20gJ2xvZGFzaCcKCmV4cG9ydCBjb25zdCBzdWIgPSBsb2Quc3Vi))

Reply:
Yes that'd be better and I've thought about it but encountered some issues to implement.
exportsInfo doesn't care about the imported name, a default value maybe imported as different names in other file, I thought the `getUsedName()` can get the used export name but it's not what I think. Maybe I can achieve this another PR